nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/src/main/java/org/apache/nifi/elasticsearch/ElasticSearchClientServiceImpl.java

```diff
@@ -790,6 +791,30 @@ private Response performRequest(final String method, final String endpoint, fina
         if (entity != null) {
             request.setEntity(entity);
         }
+
+        if (getLogger().isDebugEnabled()) {
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            entity.writeTo(out);
+            out.close();
+
+            StringBuilder builder = new StringBuilder();
```

Comment:
```suggestion
            StringBuilder builder = new StringBuilder(1000);
```

Initialising the `StringBuilder` with a large(r) buffer is sensible when we expect to be putting a lot of characters in the String - it's more performant (marginal gains, apprectiating this is only being called if DEBUG logging is enabled for the class)